packages/commonwealth/server/util/denormalizedCountUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { DB } from '@hicommonwealth/model';
+import { Transaction } from 'sequelize';
+
+export const incrementProfileCount = async (
+  models: DB,
+  community_id: string,
+  user_id: number,
+  transaction: Transaction,
+) => {
+  await models.sequelize.query(
+    `
+      UPDATE "Communities" as c
+      SET profile_count = profile_count + 1
+      WHERE c.id = :community_id
+      AND NOT EXISTS (
+        SELECT 1
+        FROM "Addresses" as a
+        WHERE a.community_id = c.id AND a.user_id = :user_id
+      );
```

Comment:
Probably was discussed already, but kinda curious why the `NOT EXISTS` condition is required?

Reply:
Hasn't been discussed yet! So this increments the profile count. Because you can join a community with multiple addresses belonging to the same profile, only the first one should count as one profile_count on the associated community.

Reply:
Ok LGTM